File: Modul-3 (Ibnu)/SOAL 2/PRAK302-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int ANGKO;
    char CONVERSIHURUF;

    printf("Input: ");
    scanf("%d", &ANGKO);

    if (ANGKO >= 80) {
        CONVERSIHURUF = 'A';
    } else if (ANGKO >= 70) {
        CONVERSIHURUF = 'B';
    } else if (ANGKO >= 60) {
        CONVERSIHURUF = 'C';
    } else if (ANGKO >= 50) {
        CONVERSIHURUF = 'D';
    } else {
        CONVERSIHURUF = 'E';
    }

    printf("Output:\n %c", CONVERSIHURUF);

    return 0;
}

File: Modul-3 (Ibnu)/SOAL 2/PRAK302-2310817210009-MuhammadIbnuSina.py
ANGKO = int(input("Input: "))

if ANGKO >= 80:
    KONVERSI_HURUF = 'A'
elif ANGKO >= 70:
    KONVERSI_HURUF = 'B'
elif ANGKO >= 60:
    KONVERSI_HURUF = 'C'
elif ANGKO >= 50:
    KONVERSI_HURUF = 'D'
else:
    KONVERSI_HURUF = 'E'

print(f"Output: {KONVERSI_HURUF}")